src/Examine.AzureDirectory/AzureLuceneDirectory.cs

```diff
@@ -13,21 +13,19 @@
 
 namespace Examine.AzureDirectory
 {
-
     /// <summary>
     /// A Lucene directory used to store master index files in blob storage and sync local files to a %temp% fast drive storage
     /// </summary>
     public class AzureLuceneDirectory : ExamineDirectory
     {
-        private readonly string _storageAccountConnectionString;
+        internal readonly string _storageAccountConnectionString;
```

Comment:
Not sure why the connection string should be internal.

Reply:
@nzdev I think it is leftover from me moving staff around